types/react/index.d.ts

```diff
@@ -128,7 +128,7 @@ declare namespace React {
         (props?: ClassAttributes<T> & P | null, ...children: ReactNode[]) => DOMElement<P, T>;
 
     // tslint:disable-next-line:no-empty-interface
-    interface HTMLFactory<T extends HTMLElement> extends DetailedHTMLFactory<AllHTMLAttributes<T>, T> {}
+    interface HTMLFactory<T extends HTMLElement> extends DetailedHTMLFactory<AllHTMLAttributes<T>, T> { }
```

Comment:
Hmm. I feel like the formatting changes are to be avoided, if only to reduce the change size?

Reply:
Agreed, this is unintentional. I'll change it.